oap-server/pom.xml

```diff
@@ -378,11 +382,75 @@
                 <artifactId>nacos-client</artifactId>
                 <version>${nacos.version}</version>
             </dependency>
+
             <dependency>
                 <groupId>org.mousio</groupId>
                 <artifactId>etcd4j</artifactId>
+                <exclusions>
+                    <exclusion>
+                        <artifactId>netty-codec-dns</artifactId>
+                        <groupId>io.netty</groupId>
+                    </exclusion>
+
+                    <exclusion>
+                        <artifactId>netty-codec-dns</artifactId>
+                        <groupId>io.netty</groupId>
+                    </exclusion>
+
+                    <exclusion>
+                        <artifactId>netty-codec-http</artifactId>
+                        <groupId>io.netty</groupId>
+                    </exclusion>
+
+                    <exclusion>
+                        <artifactId>netty-handler</artifactId>
+                        <groupId>io.netty</groupId>
+                    </exclusion>
+
+                    <exclusion>
+                        <artifactId>netty-resolver-dns</artifactId>
+                        <groupId>io.netty</groupId>
+                    </exclusion>
+
+                    <exclusion>
+                        <groupId>com.fasterxml.jackson.module</groupId>
+                        <artifactId>jackson-module-afterburner</artifactId>
+                    </exclusion>
+                </exclusions>
                 <version>${etcd4j.version}</version>
             </dependency>
+
+            <dependency>
+                <groupId>io.netty</groupId>
+                <artifactId>netty-handler</artifactId>
+                <version>${netty.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>io.netty</groupId>
+                <artifactId>netty-resolver-dns</artifactId>
+                <version>${netty.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>io.netty</groupId>
+                <artifactId>netty-codec-dns</artifactId>
+                <version>${netty.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>io.netty</groupId>
+                <artifactId>netty-codec-http</artifactId>
+                <version>${netty.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>com.fasterxml.jackson.module</groupId>
+                <artifactId>jackson-module-afterburner</artifactId>
+                <version>${jackson-module-afterburner.version}</version>
```

Comment:
Where are the licenses of these? I think you haven't updated the dist LICENSE file.

Reply:
@wu-sheng  I have added .